libraries/inverse/minimumNorm/minimumnorm.cpp

```diff
@@ -192,7 +192,7 @@ MNESourceEstimate MinimumNorm::calculateInverse(const FiffEvoked &p_fiffEvoked,
 
 //*************************************************************************************************************
 
-MNESourceEstimate MinimumNorm::calculateInverse(const MatrixXd &data, float tmin, float tstep) const
+MNESourceEstimate MinimumNorm::calculateInverse(const MatrixXd &data, float tmin, float tstep, bool pick_normal) const
```

Comment:
Is the picknormal checked with the python version?

Reply:
No testing vs mne-python was performed